js/core/polyfills/object.js

```diff
@@ -0,0 +1,35 @@
+const windowUtils = require('../../core/utils/window');
```

Comment:
/cc @nightskylark

Reply:
Copy-paste?|
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill

Reply:
Yes, this is copy-paste.
This polyfill is only needed for IE.

Reply:
1) Used `es6-object-assign` npm polyfill.
2) Created issue in RRule repo [Using RRule in IE11](https://github.com/jakubroztocil/rrule/issues/402)